.github/workflows/check-upstream-devel.yml

```diff
@@ -17,8 +17,6 @@ jobs:
     outputs:
       cl_id: ${{ steps.check_cl.outputs.id }}
       cl_rebuild: ${{ steps.check_cl.outputs.rebuild }}
-      modulo_develop_id: ${{ steps.check_modulo_develop.outputs.id }}
-      modulo_develop_rebuild: ${{ steps.check_modulo_develop.outputs.rebuild }}
       modulo_humble_id: ${{ steps.check_modulo_humble.outputs.id }}
       modulo_humble_rebuild: ${{ steps.check_modulo_humble.outputs.rebuild }}
```

Comment:
I think renaming these will also be part of the TODO you mentioned (when modulo develop branch officially supports humble). But for now it's more clear to keep them named like this.

Either it becomes `cl_develop_id` and `modulo_develop_id`, or back to just `cl_id` and `modulo_id`, whichever is cleaner. Since they are just output flags, the latter is probably better and more consistent with the other workflow.

Reply:
Indeed yeah, and good suggestion :+1: